phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerToolsTab.java

```diff
@@ -68,6 +72,27 @@ private void populate() {
         toolsContentPane.add(openTerminal, 0, 0);
         toolsContentPane.add(wineToolCaption(tr("Open a terminal")), 0, 1);
 
+        Button runExecutable = new Button();
+        runExecutable.getStyleClass().addAll("wineToolButton", "runExecutable");
+        runExecutable.setOnMouseClicked(event -> {
+            this.lockAll();
+            FileChooser fileChooser = new FileChooser();
+            fileChooser.setTitle(tr("Choose executable"));
+            File file = fileChooser.showOpenDialog(this.getContent().getScene().getWindow());
+            if (file != null) {
+                winePrefixContainerController.runInPrefix(container, file.getAbsolutePath(), this::unlockAll,
+                        e -> Platform.runLater(() -> new ErrorMessage("Error", e).show()));
+            }
+            this.unlockAll();
```

Comment:
I'm not sure it's a good idea to explicitly call this here, because you've already passed `unlockAll` as the callback to `runInPrefix`.